crates/sui-benchmark/src/bin/shared.rs

```diff
@@ -249,14 +250,12 @@ async fn main() {
                                         )))
                                     }
                                     Err(_sui_err) => {
-                                        //eprintln!("{}", sui_err);
-                                        // TODO (GAS-LEAK): How do we add this gas back in the pool?
-                                        NextOp::Response(None)
+                                        // eprintln!("{}", sui_err);
+                                        NextOp::Retry(Box::new((tx, counter_id, owner)))
                                     }
                                     _ => {
                                         // eprintln!("error");
```

Comment:
I'd nix those prints or make the logs while we're at it.

Reply:
done